sharding-core/sharding-core-parse/sharding-core-parse-mysql/src/main/java/org/apache/shardingsphere/core/parse/MySQLParser.java

```diff
@@ -23,11 +23,11 @@
 
 /**
  * SQL parser for MySQL.
- * 
+ *
  * @author duhongjun
  */
 public final class MySQLParser extends MySQLStatementParser implements SQLParser {
-    
+
```

Comment:
Please keep origin code style

Reply:
@terrymanu I have already revert MySQLParser.Please review this pull request again.